ScheduleLesson/Services/IScheduleService.cs

```diff
@@ -4,12 +4,11 @@ namespace ScheduleLesson.Services
 {
     public interface IScheduleService
     {
-        Task<List<Schedule>> GetAllSchedule();
-        Task<Schedule> GetSchedule(int id);
-        Task<Schedule> AddSchedule(Schedule schedule);
-        Task<Schedule> UpdateSchedule(Schedule schedule);
-        Task<Schedule> DeleteSchedule(int id);
-        Task<List<string>> GetDateTimeSchedule(DateTime dateTime);
-
+        Task<List<Schedule>> GetAllSchedule(Guid userId);
```

Comment:
It's better to use the name UserGuid to make it immediately clear that it's a GUID and not a numeric Id. This will make the code more readable and avoid confusion.

Ctrl +R +R - useful shortcut for renaming